packages/main/src/RPA/JavaAccessBridge.py

```diff
@@ -117,7 +111,22 @@ def __init__(
             self.visible_children = node.get_visible_children()
             self.visible_children_count = node.visible_children_count
             self.index_in_parent = node.context_info.indexInParent
-            self.column_count = column_count or len(self.visible_children)
+            self.column_count = column_count
+            self.set_rows_columns(self.visible_children, index=index)
```

Comment:
Correctly computes the number of columns a row has by checking the cell coordinates. (as soon as the next child will overlap on OX with the first one, it means we're on the 2nd row (if any))